<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start_Dialog</class>
 <widget class="QDialog" name="Start_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_prompt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose the size of the table you want to transform</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_rows">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>max value is 12</string>
   </property>
   <property name="toolTipDuration">
    <number>-2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_clmns">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>max value is 12</string>
   </property>
   <property name="toolTipDuration">
    <number>-2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_clmn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_row">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_forG">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>and the type of graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>169</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton_dir">
      <property name="text">
       <string>directed</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioButton_undir">
      <property name="text">
       <string>undirected</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_matrix">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Initial matrix</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>192</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton_inc">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Incidence</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioButton_adj">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Adjacency</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
